ext/TensorKitChainRulesCoreExt/linalg.jl

```diff
@@ -77,6 +77,12 @@ function ChainRulesCore.rrule(::typeof(adjoint), A::AbstractTensorMap)
     return adjoint(A), adjoint_pullback
 end
 
+function ChainRulesCore.rrule(::typeof(twist), A::AbstractTensorMap, is; inv::Bool=false)
+    tA = twist(A, is; inv)
+    twist_pullback(ΔA) = NoTangent(), twist(unthunk(ΔA), is; inv=!inv), NoTangent()
```

Comment:
Are you sure about the `inv != inv`? I get confused. It seems right, but we would need an actual anyonic test to be sure.

Reply:
I forgot that we're not testing this, I'll add a test tomorrow because I was definitely just guessing and hoping that it would be caught by a test :)